React framework

herecomesjaycee opened the pull request:
Reference: [Blog](https://mherman.org/blog/dockerizing-a-react-app/)

Commit:
Create React Dockerfile

Commit:
Create React Skeleton

Commit:
Update Dockerfile with the correct application folder context

GenesisSelf commented on Dockerfile:
```diff
@@ -0,0 +1,16 @@
+# base image
+FROM node:12.2.0-alpine
```
Do we want to use node 12?

mjg123 replied:
Discussed with team - I think it's good unless we find a reason not to.

GenesisSelf commented on README.md:
```diff
@@ -1 +1,68 @@
-# period-dignity-app-client
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Loving how meticulous this is 💃

GenesisSelf commented on public/robots.txt:
```diff
@@ -0,0 +1,2 @@
+# https://www.robotstxt.org/robotstxt.html
```
Do we need this file?

mjg123 replied:
I'll remove it

GenesisSelf reviewed (commented):
Just a couple of questions regarding some files and versions.

mjg123 committed:
Changes to dockerfile to add app to container

Removes "/client" references and adds app source and public files to
the container image to make it self contained.

mjg123 committed:
Removes robots.txt

Unnecessary
